python/src/functions.rs

```diff
@@ -149,13 +157,27 @@ fn md5(value: expression::Expression) -> expression::Expression {
     }
 }
 
+#[pyfunction]
+fn now(value: expression::Expression) -> expression::Expression {
+    expression::Expression {
+        expr: logical_plan::now(value.expr),
```

Comment:
Do you know why the logical `now` has one argument? I though that the physical one had the `NullArray`, but that in the logical plane, the function would be zero-argument?

Reply:
maybe because they all take one arg which is the `&[Arg]`?